blueprints/spaghetti_detection.yaml

```diff
@@ -18,13 +18,21 @@ blueprint:
     detection_frequency:
       name: Detection Frequency
       description: The detection algorithm will run in every defined seconds
-      default: 5
```

Comment:
Could you change this to "/5". Currently default value doesn't work.

Reply:
> Could you change this to "/5". Currently default value doesn't work.

Good call! Just committed.